WORKSPACE.bazel

```diff
@@ -34,6 +22,34 @@ COMMON_CXX_WARNINGS = [
     "-Wimplicit-fallthrough",
 ]
 
+# Bazel sends stdout/stderr to file so we cannot use `-fdiagnostics-color=auto`
+# https://github.com/bazelbuild/bazel/issues/6046#issuecomment-626941725
+#
+# Clang will also require libtinfo.so.5 with `-fdiagnostics-color=always`
+# https://github.com/mun-lang/mun/issues/191#issuecomment-753402659
+# https://reviews.llvm.org/D42274
+load("//tools:host_system_libraries.bzl", "host_system_libraries")
+
+host_system_libraries(
+    name = "host_system_libraries",
+    find = ["libtinfo.so.5"],
+)
+
+load("@host_system_libraries//:defs.bzl", "HOST_SYSTEM_LIBRARIES")
+
+CLANG_COLOR_FLAG = ["-fdiagnostics-color=auto" if "libinfo.so.5" in HOST_SYSTEM_LIBRARIES else ""]
```

Comment:
According to the comment, I think this should be `always` instead of `auto`.
I changed it and amended the commit. Please fix it if that's wrong.

Reply:
yep thanks for catching that